html/php-pre-challenge1/index.php

```diff
@@ -1,5 +1,12 @@
 <?php
 for ($i = 1; $i <= 100; $i++) {
-    //ここからコードを書く
-    echo $i;
+    if ($i % 15 == 0) {
```

Comment:
```suggestion
    if ($i % 15 === 0) {
```
15で割ってるのはナイスですね！
比較演算子は他の個所も全て厳密比較 `===` を使うようにしましょう

Reply:
厳密比較との違いを理解していませんでした。
３箇所とも === に置き換えました。